src/abstract.js

```diff
@@ -385,35 +374,15 @@ Sk.abstr.sequenceGetCountOf = function (seq, ob) {
 };
 
 Sk.abstr.sequenceGetItem = function (seq, i, canSuspend) {
-    var seqtypename;
-    if (seq.mp$subscript) {
-        return seq.mp$subscript(i);
-    }
-
-    seqtypename = Sk.abstr.typeName(seq);
-    throw new Sk.builtin.TypeError("'" + seqtypename + "' object is unsubscriptable");
+    return Sk.abstr.objectGetItem(seq, i, canSuspend);
```

Comment:
If we're going properly break-y, we should just remove `Sk.abstr.sequenceGetItem` and friends!

(And good riddance.)